data/templates/default.latex

```diff
@@ -156,6 +156,20 @@ $if(CJKmainfont)$
   \fi
 $endif$
 \fi
+%% Support for special Unicode characters
+% Zero width non-joiner
+\def\zerowidthnnonjoiner{%
+  \texorpdfstring{%
+    \textormath{\nobreak\discretionary{-}{}{\kern.03em}}{}%
+  }{}%
+}
+\ifPDFTeX
+  \DeclareUnicodeCharacter{200C}{{\zerowidthnnonjoiner}}
+\else
+  % the next two lines both contain an invisible zero width non-joiner char
```

Comment:
Why not using the `^^^^` notation instead?

Reply:
The engine appears to be unhappy about anything but the literal unicode char. I had tried `\char"200C`, but that doesn't work either.

Reply:
You mean that `^^^^200c` doesn't work?  If this is the case maybe `^` has been redefined?

Reply:
Oh, turns out I was using a capital `C` as required by `\DeclareUnicodeCharacter`, but had to use a lowercase `c` in this case. TeX is weird sometimes.